chain/x/oracle/handler_test.go

```diff
@@ -508,295 +508,7 @@ func TestEndBlock(t *testing.T) {
 }
 
 func TestEndBlockExecuteFailedIfExecuteGasLessThanGasUsed(t *testing.T) {
```

Comment:
These tests are removed as we are moving toward simpler EndBlock logic. I will make sure we have good tests later!